Digraph exceptions

ttasovac opened the pull request:
Проширен број изузетака за конверзију (дј, дж и нј).

ttasovac committed:
update digraphExceptions for "dj"

ttasovac committed:
update digraphExceptions for "dž"

ttasovac committed:
udpate digraphExceptions for "nj"

ttasovac committed:
fix missing comma

ttasovac committed:
small fixes

turanjanin commented on content.js:
```diff
@@ -346,35 +537,49 @@ if (window.contentScriptInjected !== true) {
-            "predživot",
+          "feldžandarm",
+          "nadždrel",
+          "nadžet", "nadžanj", "nadžm", "nadžnj", "nadžeo", "nadžel",
```
Can you move these words to new lines?

turanjanin commented on content.js:
```diff
@@ -287,36 +287,227 @@ if (window.contentScriptInjected !== true) {
+          "zasmrdje",
+          "zastidje",
+          "zaštedje",
+          "zdjel",
+          "zdjelj",
+          "zdje",
```
"zdje" already covers both the "zdjel" and "zdjelj" so we can remove these longer variants.

turanjanin reviewed (commented):
Thanks for these additions! I already received some complaints that extension doesn't deal with ijekavian words properly. It seems that this PR will cover most of the cases.

Can you just reindent the code using 4 spaces instead of 2, so we can be consistent with the rest of the file? Until I configure some linter, I need to remind people manually :)

ttasovac committed:
Update content.js

formatting (because Jovan loves four-space-tabs :smile:) + some additional fixes (removing overlaps of shorter and longer strings) + "urli" (for urlikati etc.)
